.github/workflows/ci-website.yml

```
@@ -0,0 +1,19 @@
+name: ci-website
+on:
+  push:
+    branches: [gh-pages]
+  pull_request:
+    branches: [gh-pages]
+  workflow_dispatch:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.4.0
+      - name: Setup Python
+        uses: actions/setup-python@v2.3.1
+        with:
+          python-version: 3.9
+      - name: pre-commit
+        uses: pre-commit/action@v2.0.3
```

Add a line break at the end please.
```suggestion
        uses: pre-commit/action@v2.0.3
        
```

---

I have added the file to the files checked by pre-commit, so it enforces this new line now.